src/metax_api/tests/api/rest/base/views/files/write.py

```diff
@@ -700,6 +726,26 @@ def test_update_file_allowed_projects_not_dict(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual('json' in response.data['detail'], True, 'Error regarding datatype')
 
+    def test_put_allowed_checksum_algorithm(self):
```

Comment:
I think it is not necessary to perform these same tests for each update type ``PUT`` and ``PATCH``, if ``POST`` is already tested previously above, since it should be guaranteed by other tests that the serializer's ``is_valid()`` method is always called when creating or updating files. Less tests to maintain!

Reply:
true that